spring-mvc-simple-2/src/main/java/com/baeldung/exclude_urls_filter/controller/FAQController.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.exclude_urls_filter.controller;
+
+import com.baeldung.exclude_urls_filter.service.FAQService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/faq")
+public class FAQController {
+
+    private FAQService faqService;
+
+    @Autowired
+    public FAQController(FAQService faqService) {
```

Comment:
if we are doing constructor injection let's make the var final

Reply:
Done 🙂